.github/workflows/download-data.yml

```diff
@@ -0,0 +1,44 @@
+name: download-data
+
+on:
+    schedule:
+        # Run everyday at 2:00pm CST which is 7:00pm UTC
+        - cron: 0 19 * * *
+
+    # Allows the workflow to be run manually from the Actions tab
+    workflow_dispatch:
+
+    push:
+        branches:
+            - github-action
+
+
+jobs:
+    download-data-json:
+        runs-on: macos-latest
+        steps:
+            - uses: actions/checkout@v3
+            - run: |
+                 wget https://chn-ghost-buses-public.s3.us-east-2.amazonaws.com/frontend_data_files/data.json \
```

Comment:
Currently the backend is uploading files to s3 at `cta_schedule_zipfiles_raw/google_transit_{today}.zip`. Is there a plan that the backend GitHub Action will be uploading updated json data at `frontend_data_files/data.json` and `schedule_vs_realtime_all_day_types_routes.json`? I see several backend PRs. Are there particular PRs I should focus on in the backend repo in order to understand what the files in s3 will consist of?

Reply:
Yes, `data.json` and `schedule_vs_realtime_all_day_types_routes.json` are created [here](https://github.com/chihacknight/chn-ghost-buses/blob/80fac255d898f37b6adbfcec069795607b26aadb/scrape_data/cta_data_downloads.py#L165-L186). This is in [PR #69](https://github.com/chihacknight/chn-ghost-buses/pull/69).

Reply:
Thanks for the clarification. In this case, I'm ready to approve. Shall we convert to draft first so that we don't merge before https://github.com/chihacknight/chn-ghost-buses/pull/69 is merged, or will we just remember the proper sequence of merging?

Reply:
Thanks, converting to a draft would be a good idea.